### ISSUE TITLE
Connect/Cancel buttons not fully visible

### ISSUE THREAD

**Issue by filipalai** [TabularEditor/TabularEditor#1157]
!!! Please make sure you're using the latest version of Tabular Editor 2.X before submitting a bug !!!
Same issue in v. 2.17.1 as in 2.21.1.

**Describe the bug**
See image. Not sure exactly when this started to happen, but in the last two weeks.

**To Reproduce**
1. Choose Open from DB.
2. See error.

**Expected behavior**
Buttons should be fully visible.

![TabularEditorConnect](https://github.com/TabularEditor/TabularEditor/assets/103175605/0020ec6f-0a8e-41b8-a682-e606b3c18a6f)

**Comment by otykier** [comment]
Try to set up display scaling override for Tabular Editor, by doing the following:

1. In Windows Explorer, locate TabularEditor.exe. Right-clicking on it and choose "Properties".
2. Go to the "Compatibility" tab (see screenshot below)
3. Click the "Change high DPI settings" button
4. Put a checkmark in "Override high DPI scaling behavior". Choose "System" or "System (Enhanced)" in the dropdown.
5. Click OK to close both windows.
6. Launch Tabular Editor again.

<img width="620" alt="image" src="https://github.com/TabularEditor/TabularEditor/assets/8976200/e45c84b6-88a7-47c9-8dc9-035d9f37088f">

**Comment by filipalai** [comment]
Hi, thanks for the quick response.

Tried your suggested steps, without any difference. I also tried to open that dialog box on the laptop screen, but same issue. It is like that popup window is too small for its contents.

**Comment by otykier** [comment]
Have you also tried checking the option at the top?

<img width="249" alt="image" src="https://github.com/TabularEditor/TabularEditor/assets/8976200/ce71340d-06ff-4920-b306-66dffc181757">

If this didn't help, I need some more information to try to replicate this on my end:

- Which version of Windows are you using?
- You mentioned display scaling is set to 175%. Does changing this affect how the dialog looks?
- Did you adjust the Text Size under Accessibility settings in Windows?

**Comment by filipalai** [comment]
I changed the Program DPI setting, but no difference.

I am using Windows 11.

**Yes, I turned off custom scaling (set to 175 %) and signed out. When signing back in, the problem was resolved.**

No, no Accesibility setting changed.

**Comment by otykier** [comment]
Ah right - I forgot that some applications (including TE2) require you to sign out and sign back in, after changing the display scaling. Does that mean we can close the ticket?

**Comment by filipalai** [comment]
I guess we can close it, if you are fine with the issue occurring when using custom scaling :-)

**Comment by otykier** [comment]
But didn’t you say the issue disappeared even with custom scaling, after signing out and in again?

**Comment by filipalai** [comment]
Nope. I only said that when I turned it off and signed out/in the issue was resolved. That setting was changed (set to custom scaling) earlier, and I have since that restarted the laptop multiple times.
